Update description of SENT-template (#56)
sveltejs/integrations: readme.md

- [sapper-typescript-graphql-template](https://github.com/babichjacob/sapper-typescript-graphql-template)
  - Template that includes Sapper, TypeScript preprocessing, and a GraphQL server through TypeGraphQL.
- [SENT-template](https://github.com/Zimtir/SENT-template)

>   - A small subframework with tools for full development flow like Tests, Docker, Linters, Component Library, Security, TypeScript and other tools)

Reviewer: There's an ending paren without a matching start paren and most of these words don't need to be capitalized. How about this:

```suggestion
  - A small sub-framework with tools for full development flow like tests, Docker, linting, component library, security, TypeScript, etc.
```
